azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -898,6 +916,68 @@ resource "azurerm_virtual_machine_scale_set" "test" {
 }
 `
 
+var testAccAzureRMVirtualMachineScaleSet_basicLinux_managedDiskNoName = `
```

Comment:
_minor_: for newer tests we're switching to use a function which returns a formatted string